dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -76,7 +82,21 @@ generate_configuration_with_puppet() {
     CHE_ENV_FILE="${CHE_HOST_INSTANCE}/config/$CHE_MINI_PRODUCT_NAME.env"
   fi
 
-  if [ "${CHE_DEVELOPMENT_MODE}" = "on" ]; then
+  if debug_server; then
+    DEV_MODE="development"
+    WRITE_LOGS=""
```

Comment:
does that mean that puppet logs will be shown only in debug mode? debug mode should only activate debug and do not change any other behaviour